example/typedoc.json

```diff
@@ -2,5 +2,13 @@
     "$schema": "https://typedoc.org/schema.json",
     "entryPoints": ["./src"],
     "sort": ["source-order"],
-    "media": "media"
+    "media": "media",
+    "categorizeByGroup": false,
+    "searchCategoryBoosts": {
+        "Component": 2,
+        "Model": 1.2
+    },
+    "searchGroupBoosts": {
+        "class": 1.5
```

Comment:
Note that I'm lowercasing everything, but the actual kind is "Class".  Any preference on casing?

Reply:
Speaking of which, are you sure that `searchKindBoosts` wouldn't be a better name for this? You refer to "kind" in the description for `requiredToBeDocumented`...

Reply:
With 0.22, yes, `searchKindBoosts` does look like the better name - I'm looking ahead to 0.23 though, where TypeDoc accepts an `@group` tag. By default, groups are determined by kind, but this gives additional flexibility. I think I'm happy with this, just need to find the time to pull it down and play with it a bit :)

Reply:
okie-doke